src/object-store/src/layers/lru_cache/read_cache.rs

```diff
@@ -51,9 +51,9 @@ fn can_cache(path: &str) -> bool {
     !path.ends_with("_last_checkpoint")
 }
 
-/// Generate an unique cache key for the read path and range.
-fn read_cache_key(path: &str, range: BytesRange) -> String {
-    format!("{:x}.cache-{}", md5::compute(path), range.to_header())
+/// Generate a unique cache key for the read path and range.
+fn read_cache_key(path: &str) -> String {
+    format!("{:x}.cache", md5::compute(path))
 }
```

Comment:
@Xuanwo supposes that we don't need to cache ranges but always read the whole object.

I'll comment on other diff below.

Reply:
Hi, this understanding is not true. I will give it a fix.